dotcom-rendering/cypress/integration/parallel-6/paid.content.spec.js

```diff
@@ -59,12 +52,16 @@ describe('Paid content tests', function () {
 			let requestURL = interception.request.url;
 			expect(requestURL).to.include('ec=click');
 			expect(requestURL).to.include('ea=sponsor%20logo');
-			expect(requestURL).to.include('el=british%20airways%20holidays');
+			expect(requestURL).to.include('el=ocado');
 		});
 	});
 
 	it('should send Google Analytics message on click of sponsor logo in onwards section', function () {
-		cy.visit(`Article?url=${paidContentPage}`);
+		const paidContentPage =
+			'https://www.theguardian.com/feasting-with-ocado/2022/apr/08/rukmini-iyers-recipe-for-slow-cooked-lamb-leg-al-pastor-with-pineapple-salsa';
```

Comment:
I could have used this article for the metadata test as well but that has the same `commercialProperties` across all editions but that wouldn't show line `cy.visit(`Article?url=${paidContentPage}`, { headers: { 'x-gu-geolocation': 'ip.dummytext,country:UK' }, });` is working as expected.